docs/reverse_proxy.md

```diff
@@ -182,11 +182,9 @@ matrix.example.com {
 
 ```
 frontend https
-  bind :::443 v4v6 ssl crt /etc/ssl/haproxy/ strict-sni alpn h2,http/1.1
```

Comment:
the use of a directory here was deliberate - could you explain why changing to a single file is prefererable?